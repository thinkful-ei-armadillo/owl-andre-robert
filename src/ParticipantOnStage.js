import React, { Component } from 'react';
//import './ParticipantList.css';
//import Participant from './Participant';

class StageArea extends Component {
  render() {
    return (
      <div className="stage-participant">
        <h3>{this.props.obj.name}</h3>
        <img src = {this.props.obj.avatar} alt = "avatar of user"></img>
      </div>
    );
  }
}

export default StageArea;
