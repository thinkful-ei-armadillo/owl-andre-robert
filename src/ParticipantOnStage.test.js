import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ParticipantOnStage from './ParticipantOnStage';
import { STORE } from './store';

describe('Testing stage area', () => {
  // smoke test
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ParticipantOnStage key={'adfd'} obj={STORE.participants[1]} />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  //snapshot test
  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<ParticipantOnStage key={'adfd'} obj={STORE.participants[1]} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});