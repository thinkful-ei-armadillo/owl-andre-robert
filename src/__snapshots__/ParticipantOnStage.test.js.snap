// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing stage area renders the UI as expected 1`] = `
<div
  className="stage-participant"
>
  <h3>
    Caty Flucker
  </h3>
  <img
    alt="avatar of user"
    src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
  />
</div>
`;
